aggregator/src/test/java/org/jenkinsci/plugins/workflow/WorkflowRunTest.java

```diff
@@ -232,4 +230,34 @@ public void failedToStartRun() throws Exception {
         r.assertBuildStatusSuccess(r.waitForCompletion(b));
     }
 
+    @Issue("JENKINS-29571")
+    @Test public void buildRecordAfterRename() throws Exception {
```

Comment:
I never claimed compliance with any particular IDE’s autoformat options. As for NetBeans, I could have [saved you the trouble](https://netbeans.org/bugzilla/show_bug.cgi?id=104757). If you like to use an autoformatter on newly written code, fine for you, but existing code should not be reformatted anyway, at least until we have a next generation of SCM tools which operate at the level of parse trees rather than lines of text.

As to the placement of `@CheckForNull`, _logically_ this is part of the type, which `-source 8` treats properly. Whether today’s annotations can work with that model or not does not really matter; for a human reader, it is a type annotation and should be placed to match so long as a `-source 6` compiler does not complain about it.

`@Test` is an entirely different case. That is indeed a member annotation (not just in the deficient model of `-source 6`, but logically). I keep it on the same line as the method declaration because is a determiner, not an attribute. Same for `@Override`.